config-sidecar/package.json

```diff
@@ -20,7 +23,8 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.18.0",
     "yargs": "^17.0.1",
-    "zx": "^1.14.1"
+    "zx": "^1.14.1",
```

Comment:
can we move to v7 ? I think it will not break stuff and it is good to keep up with date

Reply:
done.